site/components/pages/Pledge/lib/verifySignature.ts

```diff
@@ -7,9 +7,13 @@ interface Params {
   nonce: string;
 }
 
-export const verifySignature = (params: Params): void => {
+export const verifySignature = async (params: Params): Promise<void> => {
+  // Gnosis multisig wallets are already validated client side
+  if (params.signature === "0x") return;
```

Comment:
Oh I see, in my comment above I assumed you are using the method server side, but you are not.
Unfortunately we need to.
Our API endpoint is public, since we aren't using session tokens or api keys or anything like that. The signature verification is our access token.

The way this is written, anyone can send `{ signature: "0x", address: marcCubansAddress }` and get full write permission